packages/ella/src/dom/index.ts

```diff
@@ -913,3 +914,57 @@ export function forceFocusAndOpenKeyboard(element:HTMLElement, timeout = 0) {
     console.error('Unable to force focus input', err);
   }
 }
+
+/**
+ * This method can be used to get the Device details like browser name, user agent, os name and origin i.e desktop or mobile website.
+ *
+ * @remarks
+ * This method depends on userAgent sniffing and therefore susciptible to spoofing. Avoid detecting browsers in business impacting code
+ *
+ * @example
+ * ```
+ * console.log('Get Device Details - ',getDeviceDetails());
+ * ```
+ */
+export function getDeviceDetails() {
+  try {
+    if (typeof window !== 'undefined' && navigator != null) {
+
+      const browserName = getBrowserName();
```

Comment:
why are you using deprecated method?

Reply:
This will update separately in a different PR. Will raise it as a bug in github discussion